src/server/Document/component.jsx

```diff
@@ -72,7 +77,7 @@ const Document = ({
           </>
         )}
       </head>
-      <body>
+      <body {...ampGeoPendingAttrs}>
```

Comment:
We may want to add a cypress test for ads to ensure the AMP component is removing this and loading the ad in.